src/test/java/com/rultor/agents/req/DecryptTest.java

```diff
@@ -76,6 +77,11 @@
     );
 
     /**
+     * Test profile name.
+     */
+    private static final String TEST_PROFILE_NAME = "test/test";
```

Comment:
this is a bad idea, see http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#redundant-constants